libvast/src/system/accountant.cpp

```diff
@@ -112,8 +112,8 @@ void accountant_state::command_line_heartbeat() {
     try {
       oss.imbue(std::locale(""));
     } catch (const std::exception& e) {
-      VAST_DEBUG_ANON("accoutant failed to set locale the statistics output:",
-                      e.what());
+      VAST_DEBUG(self,
```

Comment:
Should this perhaps be a warning?

Reply:
Imho warnings should be reserved for problems that affect the user in more serious way. For example if data could not be retrieved. I don't think the statistics formatting warrants that.

Reply:
Good point.